x-pack/plugins/security_solution/public/resolver/store/ui/selectors.ts

```diff
@@ -1,30 +1,50 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { createSelector } from 'reselect';
-import { ResolverUIState } from '../../types';
-
-/**
- * id of the "current" tree node (fake-focused)
- */
-export const ariaActiveDescendant = createSelector(
-  (uiState: ResolverUIState) => uiState,
-  /* eslint-disable no-shadow */
-  ({ ariaActiveDescendant }) => {
-    return ariaActiveDescendant;
-  }
-);
-
-/**
- * id of the currently "selected" tree node
- */
-export const selectedNode = createSelector(
-  (uiState: ResolverUIState) => uiState,
-  /* eslint-disable no-shadow */
-  ({ selectedNode }: ResolverUIState) => {
-    return selectedNode;
-  }
-);
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createSelector } from 'reselect';
+import { ResolverUIState } from '../../types';
+import * as locationSearchModel from '../../models/location_search';
+
+/**
+ * id of the "current" tree node (fake-focused)
+ */
+export const ariaActiveDescendant = createSelector(
+  (uiState: ResolverUIState) => uiState,
+  /* eslint-disable no-shadow */
+  ({ ariaActiveDescendant }) => {
+    return ariaActiveDescendant;
+  }
+);
+
+/**
+ * id of the currently "selected" tree node
+ */
+export const selectedNode = createSelector(
+  (uiState: ResolverUIState) => uiState,
+  /* eslint-disable no-shadow */
+  ({ selectedNode }: ResolverUIState) => {
+    return selectedNode;
+  }
+);
+
+/**
+ * The legacy `crumbEvent` and `crumbId` parameters.
+ * @deprecated
+ */
+export const breadcrumbParameters = createSelector(
```

Comment:
This function was added. the rest of the changes (except for import statements) are white space.